.travis.yml

```diff
@@ -2,10 +2,12 @@ sudo: false
 
 language: go
 
+env:
+  global:
+  - GO15VENDOREXPERIMENT=1
```

Comment:
then we need to find a way to tell tools (godep, glide) to NOT vendor
k8s.io/contrib.  It's very surprising that a change to contrib/util would
not take effect until all the various subdirs re-vendor it.

On Tue, May 10, 2016 at 10:48 AM, Prashanth B <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/kubernetes/contrib/pull/943#discussion_r62718871>:
>
> > @@ -2,10 +2,12 @@ sudo: false
> >
> >  language: go
> >
> > +env:
> > +  global:
> > +  - GO15VENDOREXPERIMENT=1
>
> We moved from global contrib/ godeps to per project to make splitting them
> out easier, and because people complained that sub-project godep revs broke
> other projects (#374 <https://github.com/kubernetes/contrib/pull/374>)
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/contrib/pull/943/files/c77292269bc9ab2fb323be2865a4790f91eacfa8#r62718871>
>